src/pages/Notes/Notes.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from "react";
+import { useNotesContext } from "../../utils/contexts/NotesContext";
+import Note from "../../components/Note";
+import NewNote from "./NewNote";
+import "./Notes.styles.css";
+import { useTermContext } from "../../utils/contexts/TermContext";
+function Notes() {
+  const { notes, saveNotes } = useNotesContext();
+  const { term } = useTermContext();
+  const [shownNotes, setShownNotes] = useState([]);
+
+  useEffect(() => {
+    let filtered = [];
+    if (term.trim() === "") filtered = notes.filter((f) => !f.archived);
+    else
+      filtered = notes.filter(
+        (f) => !f.archived && (f.title.includes(term) || f.text.includes(term))
+      );
```

Comment:
the `filtered` variable could be defined using const and the `if/else` can be changed for a ternary

```jsx
const filtered = term.trim() === ""
  ? notes.filter(({ archived }) => !archived)
  : notes.filter(
    ({ archived, title, text }) => (
      !archived && (
        title.includes(term) || 
        text.includes(term)
      )
    )
  );
```